app.js

```diff
@@ -8,410 +8,434 @@ var slackAPI = require('slackbotapi');
 var request = require('superagent');
```

Comment:
i can't comment on `.DS_Store` since github doesn't expand it, but you should add that to `.gitignore`